orchagent/aclorch.cpp

```diff
@@ -1022,7 +1022,7 @@ bool AclTable::create()
 
     sai_attribute_t attr;
     vector<sai_attribute_t> table_attrs;
-    vector<int32_t> bpoint_list = { SAI_ACL_BIND_POINT_TYPE_PORT, SAI_ACL_BIND_POINT_TYPE_LAG };
+    vector<int32_t> bpoint_list = type == ACL_TABLE_PFCWD ? { SAI_ACL_BIND_POINT_TYPE_PORT} : { SAI_ACL_BIND_POINT_TYPE_PORT, SAI_ACL_BIND_POINT_TYPE_LAG };
```

Comment:
If LAG binding is not supported on egress, will it be better to check the stage and select the binding point?

Reply:
offline discussed with guohan, currently only brcm pfcwd use egress acl and only brcm might have egress lag bind issue. Other platforms might still be available to use egress LAG binding. So it's better to restrict port binding for PFCWD which is also by design for this use case.